examples/petsc/bp3.h

```diff
@@ -76,13 +78,30 @@ CEED_QFUNCTION(Diff)(void *ctx, const CeedInt Q,
   const CeedScalar *ug = in[0], *qd = in[1];
   CeedScalar *vg = out[0];
 
+  // Quadrature Point Loop
+  CeedPragmaOMP(simd)
   for (CeedInt i=0; i<Q; i++) {
-    const CeedScalar ug0 = ug[i+Q*0];
-    const CeedScalar ug1 = ug[i+Q*1];
-    const CeedScalar ug2 = ug[i+Q*2];
-    vg[i+Q*0] = qd[i+Q*0]*ug0 + qd[i+Q*1]*ug1 + qd[i+Q*2]*ug2;
-    vg[i+Q*1] = qd[i+Q*1]*ug0 + qd[i+Q*3]*ug1 + qd[i+Q*4]*ug2;
-    vg[i+Q*2] = qd[i+Q*2]*ug0 + qd[i+Q*4]*ug1 + qd[i+Q*5]*ug2;
-  }
+    // Read spatial derivatives of u
+    const CeedScalar uJ[3]        =  {ug[i+Q*0],
```

Comment:
`uJ`? I'm not sure what J means. Jacobian?

Reply:
Yup. It was more appropriate for BP4, where u has multiple components

Reply:
Should I call it gradu or something else?